sgv2-restapi/src/main/java/io/stargate/sgv2/restsvc/grpc/ToProtoValueCodecs.java

```diff
@@ -384,5 +386,31 @@ public UUIDCodec(String typeDesc) {
     }
   }
 
+  protected static final class TimestampCodec extends ToProtoCodecBase {
+    public TimestampCodec() {
+      super("TypeSpec.Basic.TIMESTAMP");
+    }
+
+    @Override
+    public QueryOuterClass.Value protoValueFromStrictlyTyped(Object value) {
+      if (value instanceof String) {
+        return protoValueFromStringified((String) value);
+      }
+      return cannotCoerce(value);
+    }
+
+    @Override
+    public QueryOuterClass.Value protoValueFromStringified(String value) {
+      try {
+        // TODO: this implementation requires full date/time specification including timezone
+        // we could support more flexibility in format as requested in
+        // https://github.com/stargate/stargate/issues/839
+        return Values.of(Instant.parse(value).toEpochMilli());
```

Comment:
Are we sure that this format is requested? `DateTimeFormatter.ISO_INSTANT`? Looked here https://docs.datastax.com/en/cql-oss/3.x/cql/cql_reference/timestamp_type_r.html and it seems it's fine, I was confused a bit with this section:

```
where Z is the RFC-822 4-digit time zone, expressing the time zone's difference from UTC. For example, for example the date and time of Feb 3, 2011, at 04:05:00 AM, GMT:

2011-02-03 04:05+0000
2011-02-03 04:05:00+0000
2011-02-03T04:05+0000
2011-02-03T04:05:00+0000
```

Reply:
I think this code is probably copied from SGv1 REST `Converters` or some similar source. Which may or may not conform fully to expected CQL formats.

Reply:
Correct, this comes directly from the v1 code.

I think it would be a good idea to handle all the formats supported in CQL literals. We did it in the Java driver and also in the GraphQL module (see `TimestampCoercing`).

Reply:
Perhaps we should move `TimestampCoercing` (and any similar classes) into the new `sgv2-service-common` module?

Reply:
Eventually yes; maybe defer refactoring for later as I suspect we might learn more about all kinds of things we want to move around. Although that leads to the question of whether we should track such things (another GH discussion about refactoring of value conversion functionality?)